### PR TITLE
LL-5833 - setEnv("USER_ID", ...) that we now use to retrieve user id was not done

### PR DESCRIPTION
**Pull request by machard**
https://ledgerhq.atlassian.net/browse/LL-5833

### CODE REVIEW

File: src/analytics/HookAnalytics.js [hunk 1]
```diff
@@ -1,14 +1,14 @@
 import { useEffect, useState, useCallback } from "react";
 import { start } from "./segment";
 
-const HookAnalytics = ({ store }: { store: * }) => {
+const HookAnalytics = ({ store }: { store: * }, user: *) => {
```
Comment (LFBarreto) [inline src/analytics/HookAnalytics.js:4]: should'nt this be `{ store, user }: { store: * , user: * }` ?

**Review by LFBarreto — CHANGES_REQUESTED** [review]

File: src/analytics/HookAnalytics.js (hunk 1 above)
Comment (machard) [reply src/analytics/HookAnalytics.js:4]: omg you're right

**Review by juan-cortes — APPROVED** [review]

**Review by LFBarreto — APPROVED** [review]

**Review by juan-cortes — DISMISSED** [review]

**Review by LFBarreto — DISMISSED** [review]

**Review by lruggiero-ledger — APPROVED** [review]

File: src/analytics/segment.js [hunk 2]
```diff
@@ -74,6 +74,8 @@ export const start = async (store: *, { user, created }: *) => {
   }
 
   storeInstance = store;
+  if (ANALYTICS_LOGS) console.log("analytics:user", user);
+  if (ANALYTICS_LOGS) console.log("analytics:created", created);
```
Comment (LFBarreto) [inline src/analytics/segment.js:78]: why not ? :sweat_smile: 
```javascript
if (ANALYTICS_LOGS) {
  console.log("analytics:user", user);
  console.log("analytics:created", created);
  }
  ```

**Review by LFBarreto — CHANGES_REQUESTED** [review]

File: src/analytics/segment.js (hunk 2 above)
Comment (juan-cortes) [reply src/analytics/segment.js:78]:
```suggestion
  if (ANALYTICS_LOGS) {
    console.log("analytics:user", user);
    console.log("analytics:created", created);
  }
```

**Review by LFBarreto — APPROVED** [review]

**Review by juan-cortes — CHANGES_REQUESTED** [review]
⬇️

File: src/analytics/segment.js [hunk 3]
```diff
@@ -74,8 +73,12 @@ export const start = async (store: *) => {
     });
   }
 
-  const { user, created } = await getOrCreateUser();
   storeInstance = store;
+  if (ANALYTICS_LOGS) console.log("analytics:user", user);
```
Comment (juan-cortes) [inline src/analytics/segment.js:77]: I meant here 🤦🏼

**Review by juan-cortes — CHANGES_REQUESTED** [review]

**Review by juan-cortes — APPROVED** [review]

**Review by LFBarreto — APPROVED** [review]

**Pull request closed without merging**